sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -119,6 +120,8 @@ export default defineComponent({
     const isFirstPage = ref(true);
     const isLastPage = ref(false);
     const showNav = ref(false);
+    const formData = ref({} as StudentApplicationFormData);
```

Comment:
For my understanding, what is the justification for creating a new reactive variable?